photon-server/src/main/java/org/photonvision/Main.java

```diff
@@ -317,6 +318,12 @@ public static void main(String[] args) {
             logger.error("Failed to load libcamera-JNI!", e);
         }
 
+        try {
+            MrCalJNILoader.forceLoad();
+        } catch (IOException e) {
+            logger.error("Failed to load mrcal-JNI! Camera calibration will fall back to opencv", e);
```

Comment:
This should probably be a warn not an error as this could be expected behavior on certain platforms. You could also pick the level based on the platform.